test/widgets/login_test.dart

```diff
@@ -142,4 +144,46 @@ void main() {
     // TODO test handling failure in fetchApiKey request
     // TODO test _inProgress logic
   });
+
+  group('Server URL Helper Text', () {
+    Future<void> prepareAddAccountPage(WidgetTester tester) async {
+      await tester.pumpWidget(const MaterialApp(
+        localizationsDelegates: ZulipLocalizations.localizationsDelegates,
+        supportedLocales: ZulipLocalizations.supportedLocales,
+        home: AddAccountPage(),
+      ));
+    }
+
+    final zulipLocalizations = GlobalLocalizations.zulipLocalizations;
+    const url = 'https://zulip.com/help/logging-in#find-the-zulip-log-in-url';
+
+    Future<Finder> findHelperText(WidgetTester tester) async {
+      return find.byWidgetPredicate((widget) =>
+          widget is GestureDetector &&
+          widget.child is Text &&
+          (widget.child as Text).data == zulipLocalizations.serverURLDocLinkLabel);
+    }
+
+    testWidgets('launches URL when helper text is tapped', (WidgetTester tester) async {
```

Comment:
nit: follow style of existing test cases, leaving off "WidgetTester " on test-body callback:
```suggestion
    testWidgets('launches URL when helper text is tapped', (tester) async {
```